sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -23,7 +23,6 @@
 #include "sw/device/silicon_creator/lib/drivers/rstmgr.h"
 #include "sw/device/silicon_creator/lib/drivers/uart.h"
 #include "sw/device/silicon_creator/lib/error.h"
-#include "sw/device/silicon_creator/lib/log.h"
```

Comment:
We already use `-ffunction-sections` but should we also remove the log library from the dependencies of mask_rom?

Reply:
Will do.

Reply:
Done.